models/snkrsDao.js

```diff
@@ -0,0 +1,221 @@
+import { PrismaClient } from '@prisma/client';
+
+const prisma = new PrismaClient();
+
+const updataOpenClose = async (bool, style_code) => {
+  return await prisma.$queryRaw`
+    UPDATE
```

Comment:
```suggestion

const updataOpenClose = async (style_code, bool) => {
  return await prisma.$queryRaw`
    UPDATE
```
parameter가운데 함수가 적용되는 대상을 먼저 기입하시는 것이 자연스러울 것 같습니다.